<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Monkey Island SE Series: Repacker Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/iconsimple.ico</normaloff>:/icons/iconsimple.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Selected Game:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Original Pak File:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="origPakFileNameTxtBx">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="BrowseOrigPakBtn">
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Extracted Folder:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="extractedFolderNameTxtBx">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="BrowseExtrFolderBtn">
      <property name="toolTip">
       <string>Backups the current target file in the target directory. It won't bakup unsubmitted changes.</string>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="SubmitRepackBtn">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QComboBox" name="selGameCmBx">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="resetBtn">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit_3"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Original_File">
   <property name="text">
    <string>Open Original File...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>Show info about this application...</string>
   </property>
  </action>
  <action name="actionSave_Translation_As">
   <property name="text">
    <string>Save Translation Session As...</string>
   </property>
  </action>
  <action name="actionSave_Translation">
   <property name="text">
    <string>Save Translation Session</string>
   </property>
  </action>
  <action name="actionExtract_Translation_to_txt">
   <property name="text">
    <string>Extract Translation to .txt</string>
   </property>
  </action>
  <action name="actionImport_Translation_from_txt">
   <property name="text">
    <string>Import Translation from .txt</string>
   </property>
  </action>
  <action name="actionQuit_3">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Translation_File">
   <property name="text">
    <string>Load Translation in Session...</string>
   </property>
   <property name="toolTip">
    <string>Load a Translation in the active Session from a backup or old native file</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExtract_Original_to_txt">
   <property name="text">
    <string>Extract Original to .txt</string>
   </property>
  </action>
  <action name="actionMerge_Active_Translation_with">
   <property name="text">
    <string>Merge Active Translation with...</string>
   </property>
   <property name="toolTip">
    <string>Merges the translation in the active session with an external one</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionFind_Next_Marked">
   <property name="text">
    <string>Find Next Marked</string>
   </property>
  </action>
  <action name="actionFind_Previous_Marked">
   <property name="text">
    <string>Find Previous Marked</string>
   </property>
  </action>
  <action name="actionFind_Next_Conflicted">
   <property name="text">
    <string>Find Next Conflicted</string>
   </property>
  </action>
  <action name="actionFind_Previous_Conflicted">
   <property name="text">
    <string>Find Previous Conflicted</string>
   </property>
  </action>
  <action name="actionFind_Next_Unchanged">
   <property name="text">
    <string>Find Next Unchanged</string>
   </property>
  </action>
  <action name="actionFind_Previous_Unchanged">
   <property name="text">
    <string>Find Previous Unchanged</string>
   </property>
  </action>
  <action name="actionFind_Next_Changed">
   <property name="text">
    <string>Find Next Changed</string>
   </property>
  </action>
  <action name="actionFind_Previous_Changed">
   <property name="text">
    <string>Find Previous Changed</string>
   </property>
  </action>
  <action name="actionFont_crafting">
   <property name="text">
    <string>Font crafting</string>
   </property>
  </action>
  <action name="actionRepacker">
   <property name="text">
    <string>Repacker</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
